.github/workflows/go.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches:
+      - "*"
```

Comment:
I don't really know the details of GitHub actions limits for teams, but this won't give us issues, right?

Reply:
Generally shoudln't, but you're right it may be an overkill to run for every commit in all branches. Restricting to run only on "main" branch.